ani-cli

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-version_number="4.6.2"
+version_number="4.6.3"
```

Comment:
Since this is a feature update, it could be a version bump to 4.7

Reply:
done :)